packages/sources/svelte/src/components/forms/VtmnTextInput/test/VtmnTextInput.spec.js

```diff
@@ -46,6 +46,17 @@ describe('VtmnTextInput', () => {
       expect(onChange.mock.results[0].value).toEqual(nextValue);
     });
 
+    test('Should trigger focus event if focused', async () => {
```

Comment:
Thanks for your PR :) 
You can remove the async ^^

Reply:
Done ! :)